File: 0268-missing-number/0268-missing-number.cpp
class Solution {
public:
    int missingNumber(vector<int>& nums) {
        int l=nums.size();
        int s=(l*(l+1))/2;
        int sum=0;
        for(int i=0;i<l;i++)
        {
            sum=sum+nums[i];
        }
        int k=s-sum;
        return k;
    }
};

File: 0268-missing-number/0268-missing-number.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var missingNumber = function(nums) {
    var length=nums.length;
    var sum=(length*(length+1))/2;
    var s=0;
    for(var i=0;i<length;i++)
    {
        s=s+nums[i];
    }
    return sum-s;
};